config/dev.webpack.config.js

```diff
@@ -10,6 +11,10 @@ const { config: webpackConfig, plugins } = config({
   ...(process.env.BETA && { deployment: 'beta/apps' })
 });
 
+const envFileName = process.env.NODE_ENV ? `.env.${process.env.NODE_ENV}` : '.env';
```

Comment:
In doing quick local testing, it doesn't seem like the app cares if you have a default value here.  Could we just have:
```ts
const envFileName = `.env.${process.env.NODE_ENV}`;
```
I at least I don't see any errors if the file doesn't exist.

Reply:
Or maybe just make the default value empty string or null?

Reply:
ah ok, so in the package.json it was setting the NODE_ENV=development, so with this it was resolving to .env.development which we just deleted, so that was coming back empty and then just defaulting to the .env values.  

I just pushed a change that will set the file name as `.env`, and then if process.env.NODE_ENV is there and isn't 'development' it will set it to `.env.${process.env.NODE_ENV}`.  This makes sense to me and can be extended (e.g., if we add a stage .env), but lmk what you think.

Reply:
I guess it's fine.  It just seems odd to need any value here when we are relying on .env because that's the default anyway per this line of code:
```ts
plugins.push(new Dotenv({ path: envPath, defaults: resolve(__dirname, '../.env') }));
```
Seems like we shouldn't even need that path arg if it's the same as the defaults arg.

Reply:
ok just pushed a fix to not include the path in there if in development mode.